aptos-move/aptos-transactional-test-harness/src/aptos_test_harness.rs

```diff
@@ -1031,7 +1031,7 @@ impl<'a> MoveTestAdapter<'a> for AptosTestAdapter<'a> {
     fn deserialize(&self, bytes: &[u8], layout: &MoveTypeLayout) -> Option<Value> {
         let environment = AptosEnvironment::new(&self.storage);
         let code_storage = self.storage.as_aptos_code_storage(&environment);
-        ValueSerDeContext::new()
+        ValueSerDeContext::new(None)
```

Comment:
Can't we get `max_value_nest_depth` here?

Reply:
We can, fixed